jobs/rocoto/anal.sh

```diff
@@ -4,9 +4,36 @@ source "${HOMEgfs}/ush/preamble.sh"
 
 ###############################################################
 # Source FV3GFS workflow modules
-. ${HOMEgfs}/ush/load_fv3gfs_modules.sh
-status=$?
-[[ ${status} -ne 0 ]] && exit ${status}
+#. ${HOMEgfs}/ush/load_fv3gfs_modules.sh
+#status=$?
+#[[ ${status} -ne 0 ]] && exit ${status}
+
+# TODO: clean this up
+#Source appropriate modulefiles based on machine
+
+source "${HOMEgfs}/ush/detect_machine.sh"
+if [[ "${MACHINE_ID}" = "wcoss2" ]]; then
+    # Source FV3GFS workflow modules
+    . "${HOMEgfs}"/ush/load_fv3gfs_modules.sh
+    status=$?
+    [[ ${status} -ne 0 ]] && exit "${status}"
+else
+    # Append compiler (only on machines that have multiple compilers)
+    COMPILER=${COMPILER:-"intel"}
+    if [[ "${MACHINE_ID}" = "hera" ]] || [[ "${MACHINE_ID}" = "cheyenne" ]]; then
+	MACHINE_ID=${MACHINE_ID}.${COMPILER}
```

Comment:
```suggestion
        MACHINE_ID=${MACHINE_ID}.${COMPILER}
```

Repeat for the other files to fix the indentation.

Reply:
Indention fixed at 2d4d020f.